src/Components/SearchCollections/SearchCollections.js

```diff
@@ -42,19 +50,45 @@ class SearchCollections extends Component {
 
 	componentDidMount() {
 		this.searchCollections();
+		setTimeout(() => {
+			if (!this.state.total || this.state.total <= 15) {
+				this.setState({ isLoading: false, hasMore: false, hasError: false });
+				return;
+			}
+		}, 3000);
```

Comment:
wait for 3 sec and if the fetching result is 0 set hasMore,isLoading,hasError false